### DISCUSSION TITLE
Should I use the same DB table as Next-auth?

### DISCUSSION THREAD

**Discussion by VictorLandim** [nextauthjs/next-auth#11442] [Help]
Hey everyone. When using Next-Auth with an adapter, what is the recommended practice: 
- use the same table as next-auth to store my users' info.
- create and use a separate table instead.

Thanks!

**Comment by k-taro56** [comment]
I think either would be fine!

**Comment by DennisSpek** [comment]
Yes, I am also using the same tables that next-auth uses.

**Reply by k-taro56** [reply]
me too
